PowerShell/JumpCloud Commands Gallery/commands.json

```diff
@@ -97,6 +97,13 @@
     "description": "This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed.\n\nIn order to use this command, follow the instructions from the [Installing the CrowdStrike Falcon Agent KB](https://support.jumpcloud.com/s/article/Installing-the-Crowdstrike-Falcon-Agent)\n\nSpecifically for this command:\n\n1. Download the CrowdStrike Falcon Agent installer and host it at a URL that your devices can access. Set this URL to the \"DownloadURL\" of this script.\n2. Extend the command timeout to a value that makes sense in your environment. The suggested command timeout for an environment with average network speeds on devices with average computing power is 10 minutes. Note that the command may timeout with a 124 error code in the command result window if not extended, but the script will continue to run.\n3. It is recommended to set the Launch Event for the command to \u201cRun As Repeating\u201d with an interval of one hour."
   },
+  {
+    "name": "Mac - Install Remote Assist | v1.0 JCCG",
+    "type": "mac",
+    "command": "#!/usr/bin/env bash\nset -euo pipefail\n\ndeclare -r REMOTE_PKG_URL=\"https://cdn.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-remote-assist.pkg\"\ndeclare -r LOCAL_PKG_TMP_PATH=\"$(mktemp -d)/jumpcloud-remote-assist.pkg\"\n\nfunction get_app_pid() {\n    local -r APP_NAME=$1\n    ps -eo pid,comm | awk \"/${APP_NAME}$/ {print \\$1}\"\n}\n\nfunction kill_app_by_name() {\n    local -r APP_NAME=$1\n    if PID=$(get_app_pid \"${APP_NAME}\") && [[ -n \"${PID}\" ]]; then\n        kill -s TERM \"${PID}\"\n        sleep 1\n    fi\n\n    if PID=$(get_app_pid \"${APP_NAME}\") && [[ -n \"${PID}\" ]]; then\n        kill -s KILL \"${PID}\"\n    fi\n}\n\n# Clean up installs having legacy names\nkill_app_by_name \"Jumpcloud Assist App\"\n\necho \"Downloading JumpCloud Remote Assist installer\"\ncurl --silent --output \"${LOCAL_PKG_TMP_PATH}\" \"${REMOTE_PKG_URL}\" >/dev/null\necho \"Download complete\"\n\n( # Run in a subshell to ensure cleanup\nset +e\nSIGNATURE_OUT=$(pkgutil --check-signature \"${LOCAL_PKG_TMP_PATH}\")\n\nif [[ $? != 0 ]] ; then\n    echo \"Installer lacks a valid signature, aborting installation\"\n    exit 1\nfi\n\nif ! echo \"${SIGNATURE_OUT}\" | grep \"1. Developer ID Installer: JUMPCLOUD INC. (N985MXSH85)\" &> /dev/null; then\n    echo \"Installer lacks a valid signature, aborting installation\"\n    exit 1\nfi\n\necho \"Installing JumpCloud Remote Assist\"\ninstaller -pkg \"${LOCAL_PKG_TMP_PATH}\" -target /\necho \"Installation finished with exit code $?\"\n)\nrm -f \"${LOCAL_PKG_TMP_PATH}\"\nrmdir \"$(dirname \"${LOCAL_PKG_TMP_PATH}\")\"",
+    "link": "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Remote%20Assist.md",
+    "description": "Downloads and installs the JumpCloud Remote Assist app on a Mac system."
+  },
```

Comment:
I'm not sure we want these in commands.json. It might mean they are published to the commands gallery for all admins, which we wouldn't want for EA, we would just want select admins to know about these commands.

Reply:
These are auto-generated by the CI, so I guess if we don't want this for all to see we can fake failure like what we just had :D

Reply:
How do we use this commands gallery though?  How do we communicate it to the customers?

Reply:
Oh, I must have been mistaken then. I don't know what this repo does, sorry.

Reply:
@pdasjc What do you think?

Reply:
Per the [remote assist EA guide](https://docs.google.com/document/d/1M0IAK8gj21umsNseyysDq-LB1ubeoDb4FLvYwFJW8pg/edit), the admin is instructed to copy the commands right from the `Command` section in the support repo.  Hence, this PR effectively has no effect either way.  I'm still not sure how `PowerShell/JumpCloud Commands Gallery/commands.json` is being used but it seems safe to close this PR without merge, since we don't need the commands.json file updated for remote assist EA.